points/activities/data_selecta.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+
+#
+# This program requires to launch data from https://archive-beta.ics.uci.edu/ml/datasets/daily+and+sports+activities
```

Comment:
s/launch/load/ ? :rocket: 
(+ broken URL)

Reply:
Done on [36b0782](https://github.com/GUDHI/gudhi-data/pull/7/commits/36b0782396de99795894102ce0410245498115c5). Just hope the url won't change to often...